test/test_jit.py

```diff
@@ -5121,7 +5115,7 @@ def func_float_int(a, b):
         def divmod_test_iterator(func, num, den):
             for i in num:
                 for j in den:
-                    self.checkScript(func, (i, j))
+                    self.checkScript(func, (i, j), frames_up=2)
```

Comment:
It's odd that this now has to manually pass in the frame when it didn't before. Was the default 2 prior and now it's 1?

Reply:
It's nested in another function so we need to go up an extra frame to evaluate the function with the string frontend (which entails creating a Python function from a code string, which needs to look at the stack to get the locals/globals/closed over vars)